src/modules_manifest.ts

```diff
@@ -0,0 +1,32 @@
+export interface FileMap<T> { [fileName: string]: T; }
```

Comment:
In tsickle we now can use ES6 Map<> types.  It would require changing the code below to use map.set(key, val) and map.get(key) and so on.  Maybe for another change.